### PR TITLE
Fix search_query. Only apply camel_casing to key-term

### PR DESCRIPTION
**Pull request by dmarjenburgh**
Selectively applies camel_case conversion to key-terms in tsv search queries.

This allows searching for `productType:X` instead of `product_type:X`, but prevents arbitrary conversion of e.g. `L2V` to `l2_v`.

### CODE REVIEW

File: orchestrator/utils/search_query.py [hunk 1]
```diff
@@ -97,6 +97,11 @@ def lex(self) -> Iterable[Lexeme]:
         yield Token.END,
 
 
+def camel_to_snake(s: str) -> str:
```
Comment (mrijk) [inline orchestrator/utils/search_query.py:100]: I think we now have this function in multiple locations

**Review by mrijk — CHANGES_REQUESTED** [review]
I think we can get rid of multiple implementations of snake_to_camel

**Review by tjeerddie — APPROVED** [review]

File: orchestrator/utils/search_query.py (hunk 1 above)
Comment (dmarjenburgh) [reply orchestrator/utils/search_query.py]: Moved to utils.helpers

**Review by mrijk — APPROVED** [review]
LGTM

**Pull request merged**